src/specs/models/api-body-schemas/Project.v1.yaml

```diff
@@ -0,0 +1,28 @@
+type: object
+title: ''
+description: Schema for a sample object
+properties:
+  name:
+    type: string
+  description:
+    type: string
+  createdDate:
+    type: string
+  lastModified:
+    type: string
+  uuid:
```

Comment:
I would call this `projectUuid` to make it less ambigious

Reply:
As things currently are, `uuid` for an object (e.g. project or samples) refer to itself, so we wouldn't be doing `project.projectUuid` or `samples.sampleUuid`.